src/main/java/com/epam/aidial/cfg/domain/model/source/ModelEndpointsSource.java

```diff
@@ -0,0 +1,4 @@
+package com.epam.aidial.cfg.domain.model.source;
+
+public class ModelEndpointsSource extends ModelSource {
+}
```

Comment:
if endpoint'd be inside it'll be more consistent with other type of ModelSource

Reply:
We discussed this point last week and decided to do it like this. For container source we get base endpoint from container, concatenate it with completionEndpointPath and save it in model.endpoint. And for endpoints source we just use it as a marker and use model.endpoint. There is no point in moving endpoint from model to source or duplicating endpoint here and there

Reply:
deplicating made different sources in sync, you use source for changing and model.endpoint as materialized view. it's simplier than complex rule described above. but i won't insist here